### PR TITLE
#244 allow exoframe rm with authentication token

### PR DESCRIPTION
**Pull request by SeaLife**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,5 @@ exoframe-win.exe
 .idea/
 .DS_Store
 
+package-lock.json
+exoframe.json
```
Comment (yamalight) [inline .gitignore]: Please add EOL here

File: src/commands/remove.js [hunk 2]
```diff
@@ -7,20 +7,34 @@ const {userConfig, isLoggedIn, logout} = require('../config');
...
 
   console.log(chalk.bold('Removing deployment:'), id);
 
   // services request url
   const remoteUrl = `${userConfig.endpoint}/remove/${encodeURIComponent(id)}`;
+  let authToken = userConfig.token;
+
+  if (deployToken) {
+    authToken = deployToken;
+    console.log('\nDeploying using given token..');
```
Comment (yamalight) [inline src/commands/remove.js]: Please fix the logging message here (we're not deploying, are we?)

Comment (SeaLife) [reply src/commands/remove.js]: Is using "Undeploying ..." ok ?

File: .gitignore (hunk 1 above)
Comment (SeaLife) [reply .gitignore]: What do you mean by EOL, an empty line?

File: src/commands/remove.js (hunk 2 above)
Comment (yamalight) [reply src/commands/remove.js]: `Removing` is the terminology that was used so far, I'd prefer to be consistent

File: .gitignore (hunk 1 above)
Comment (yamalight) [reply .gitignore]: Yup

**Comment by SeaLife** [comment]
Test-Cases added.

File: src/commands/remove.js [hunk 3]
```diff
@@ -7,20 +7,34 @@ const {userConfig, isLoggedIn, logout} = require('../config');
...
+    alias: 't',
+    description: 'Deployment token to be used for authentication',
+  },
+};
+exports.handler = async (args = {}) => {
+  const deployToken = args.token;
+  const id = args.id;
+
+  if (!deployToken && !isLoggedIn()) {
     return;
```
Comment (yamalight) [inline src/commands/remove.js]: There should be an error message here notifying user that he needs to auth first.

Comment (SeaLife) [reply src/commands/remove.js]: I used the same logic, deploy.js is using... Im adding this :)

Comment (yamalight) [reply src/commands/remove.js]: Does `deploy.js` also not have an error message? 😮
If yes, I'd need to fix this 🤔

Comment (SeaLife) [reply src/commands/remove.js]: Yeah, it doesnt. There you go, there is now a error message :)

**Comment by yamalight** [comment]
Looks great, thank you!

**Pull request merged**